src/pages/extending-agent-app/products-sdk/helpdesk-sdk/index.mdx

```diff
@@ -0,0 +1,538 @@
+
+Modify the Ticket Details widget's state
+
+<CodeSample>
+
+```js
+widget
+  .modifySection({
+    title: "My widget",
+    components: [
+      {
+        type: "title",
+        data: {
+          value: "The title will be modified"
+        }
+      }
+    ]
+  })
+  .then(() => {
+    // the widget should be modified now
+  });
+```
+
+</CodeSample>
+
+## Fullscreen widgets
+
+If you want to connect a Fullscreen widget to the HelpDesk App, you should use the `createFullscreenWidget` function. It returns a promise resolving to a Fullscreen widget instance.
+
+<CodeSample>
+
+```js
+import { createFullscreenWidget } from "@livechat/helpdesk-sdk";
+
+createFullscreenWidget().then(widget => {
+  // build your logic around the widget
+});
+```
+
+</CodeSample>
+
+### Events
+
+This widget currently does not support any events.
+
+### Methods
+
+#### Set notification badge
+
+If you want to notify Agents there’s something important inside the widget, use the `setNotificationBadge` method.
+
+<CodeSample>
+
+```js
+widget.setNotificationBadge(26);
+```
+
+</CodeSample>
+
+It displays a notification badge on top of your app’s icon:
+
+<p style={{maxWidth: "70px"}}>
+  <img src="/images/extending-agent-app/fullscreen-widget-badge.png" alt="Fullscreen widget badge" width="70px" height="190px" />
+</p>
+
+If you want to remove the badge, invoke the method with the `null` parameter.
+
+<CodeSample>
+
+```js
+widget.setNotificationBadge(null);
+```
+
+</CodeSample>
+
+Make sure Agents can dismiss the notification to avoid cluttered UI.
+
+# Advanced use
+
+## Developing your own widget
+
+If you're building your own widget and you need to interact with the HelpDesk App, be sure to use the [HelpDesk SDK](#top).
+
+## Accessing HelpDesk data
+
+You can leverage the OAuth 2.1 authorization flow to use <a href="https://api.helpdesk.com/docs" target="_blank">HelpDesk API</a>. Head to [Sign in with LiveChat](/getting-started/authorization/sign-in-with-livechat/) docs for more information.
+
+## Layout and Styling
+
+If you're using React, you can use the [LiveChat Design System](/extending-ui/extending-agent-app/design-system/), which is a component library used directly in the LiveChat user interface.
+
+## Hosting the widget
+
+You can host your widget locally or on a dedicated server. The hosted content has to be served over the **HTTPS Protocol**.
+
+During development, you can use a self-signed certificate for `localhost` or upload your widget to an SSL-enabled host. You can also leverage bundlers like [Webpack](https://webpack.js.org/configuration/dev-server/) to use an https-enabled development server.
+
+If you need a quick way to **deploy** your widget, you can use [Netlify Drop](https://developers.livechat.com/blog/widget-hosting-on-netlify/).
```

Comment:
Updated address: https://developers.livechat.com/updates/widget-hosting-on-netlify/
This article is quite old. Once Oliwia's video about App Deployment is released, we can link it here instead of the article